app/gui/view/graph-editor/src/lib.rs

```diff
@@ -2596,6 +2612,59 @@ fn new_graph_editor(app: &Application) -> GraphEditor {
     }
 
 
+    // === Edited node growth/shrink animation ===
```

Comment:
This logic should be outsourced to a separate file to avoid further growing the graph editor.

Reply:
I agree, moved this FRP code to a separate module in `component/node` hierarchy (where I feel it belongs).